src/js/hooks/cycleCount/useResolveStepValidation.js

```diff
@@ -141,6 +141,16 @@ const useResolveStepValidation = ({ tableData }) => {
     isValid.current = null;
   };
 
+  /**
+   * Returns true if any of the counts in the batch have no items associated with them for the
+   * current count index. This is possible if the product had its quantities zeroed out while
+   * the count was in progress via a different process (such as record stock).
+   */
+  const anyCountHasNoItems = () => _.some(
```

Comment:
I'm just realizing that this should check only the items of the most recent count index (we could hard code this to "1" if that's easier)